types/bool.go

```diff
@@ -30,29 +31,44 @@ var _ attr.Value = Bool{}
 
 // Bool represents a boolean value.
 type Bool struct {
-	// Unknown will be true if the value is not yet known.
-	Unknown bool
+	// unknown will be true if the value is not yet known.
+	unknown bool
 
-	// Null will be true if the value was not set, or was explicitly set to
+	// null will be true if the value was not set, or was explicitly set to
 	// null.
-	Null bool
+	null bool
 
-	// Value contains the set value, as long as Unknown and Null are both
+	// Value contains the set value, as long as unknown and null are both
 	// false.
 	Value bool
 }
 
+// NewBool returns a Bool initialized with the given value.
+func NewBool(value bool) Bool {
```

Comment:
Nit: In terms of naming, do we want to consider `NewBoolUnknown()`, `NewBoolNull()` or `BoolUnknown()`, `BoolNull()`?

Reply:
I think we should treat any of these potential helpers as a separate proposal issue, so we can hash through the details separate from potentially changing the `attr.Value` interface.

Reply:
Yeah, the need arised primarily from the fact that I was renaming `Unknown` to `unknown` etc.

I can punt on that.